src/components/Reactions/Reactions.tsx

```diff
@@ -46,6 +46,13 @@ export interface ReactionsProps extends Pick<PaletteProps, 'size'>, QAProps, DOM
      * Reactions' readonly state (when a user is unable to react for some reason).
      */
     readOnly?: boolean;
+    /**
+     * Position of the "Add reaction" button.
+     * Use 'none' to hide the button.
+     *
+     * @default 'right'
+     */
+    addButtonPosition?: 'left' | 'right' | 'none';
```

Comment:
Let's separate hiding and placement, also we use logical values (start/end) instead of physical ones (left/right)

```suggestion
    addButtonPlacement?: 'start' | 'end';
```

`none` value is the same as `readOnly`, isn't it?

Reply:
- Turned `left/right` to `start/end`
- Moved `none` to a separate property `hideAddButton`

> `none` value is the same as `readOnly`, isn't it?

Not quite. You can still toggle reactions, it's just the button that hides